shell/platform/windows/flutter_windows.cc

```diff
@@ -255,3 +255,28 @@ void FlutterDesktopMessengerSetCallback(FlutterDesktopMessengerRef messenger,
                                         void* user_data) {
   messenger->dispatcher->SetMessageCallback(channel, callback, user_data);
 }
+
+FlutterDesktopTextureRegistrarRef FlutterDesktopRegistrarGetTextureRegistrar(
```

Comment:
I would expect there to also be a `FlutterDesktopEngineGetTextureRegistrar`, and corresponding accessor in the C++ wrapper.